cmd/kubectl_token.go

```diff
@@ -556,7 +556,7 @@ func getAuthProviders(server string) ([]TypedProvider, error) {
 
 			err = json.Unmarshal([]byte(provider.Raw), typedProvider)
 			if err != nil {
-				return nil, err
+				return nil, fmt.Errorf("attempting to decode the auth provider of type %s", providerType)
```

Comment:
Sorry, my fault, this should of course include the error...

Reply:
Wops, make sense!